remove.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+# Initialize basic vars and helpers
+source blueprint-lib/init.sh
+
+##
+# Add dependencies here
+##
+
+# source blueprint-lib/docker.sh
+
+##
+# ADD BLUEPRINT CODE BELOW HERE
+#
+# BASE_PATH is the full path to the project root
+# APP_NAME is the name of the Django app that will be modified
+##
+
+BLUEPRINT="MapsBlueprint"
```

Comment:
This should be `Maps`:
https://github.com/crowdbotics/blueprint-react-native-maps-screen/blob/master/install.sh#L19

Also please remove the duplicate variable after that.